client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -0,0 +1,369 @@
+							{ translate( 'City of Birth' ) }
+						</FormLabel>
+						<FormTextInput
+							id="placeOfBirth"
+							value={ placeOfBirth }
+							placeholder={ translate( 'City of birth' ) }
+							onChange={ this.handleChangeEvent } />
+					</FormFieldset>
+				) }
+
+				{ countryOfBirth === 'FR' && (
+					<FormFieldset>
+						<FormLabel htmlFor="postalCodeOfBirth">
+							{ translate( 'Postal Code of Birth' ) }
+						</FormLabel>
+						<FormTextInput
+							id="postalCodeOfBirth"
+							value={ postalCodeOfBirth }
+							type="number"
+							autoCapitalize="off"
+							autoComplete="off"
+							autoCorrect="off"
+							placeholder={ translate( 'ex. 75008' ) }
+							onChange={ this.handleChangeEvent } />
+					</FormFieldset>
+				) }
+			</div>
+		);
+	}
+
+	renderOrganizationFields = () => {
+		const translate = this.props.translate;
+		const {
+			registrantVatId,
+			sirenSiret,
+			trademarkNumber
+		} = { ...emptyValues, ...this.props.values };
+		return (
+			<div>
+				<FormFieldset>
+					<FormLabel className="registrant-extra-info__optional"
+						htmlFor="registrantVatId">
+						{ translate( 'VAT Number' ) }
+						{ this.renderOptional() }
+					</FormLabel>
+					<FormTextInput
+						id="registrantVatId"
+						value={ registrantVatId }
+						autoCapitalize="off"
+						autoComplete="off"
+						autoCorrect="off"
+						placeholder={ translate( 'ex. XX123456789' ) }
+						onChange={ this.handleChangeEvent } />
+				</FormFieldset>
+
+				<FormFieldset>
+					<FormLabel className="registrant-extra-info__optional"
+						htmlFor="sirenSiret">
+						{ translate( 'SIREN or SIRET Number' ) }
+						{ this.renderOptional() }
+					</FormLabel>
+					<FormTextInput
+						id="sirenSiret"
+						value={ sirenSiret }
+						placeholder={
+							translate( 'ex. 123 456 789 or 123 456 789 01234',
+								{ comment: 'ex is short for "example". The numbers are examples of the EU VAT format' }
+							)
+						}
+						autoCapitalize="off"
+						autoComplete="off"
+						autoCorrect="off"
+						onChange={ this.handleChangeEvent } />
+				</FormFieldset>
+
+				<FormFieldset>
+					<FormLabel className="registrant-extra-info__optional"
+						htmlFor="trademarkNumber">
+						{ translate( 'EU Trademark Number' ) }
+						{ this.renderOptional() }
+					</FormLabel>
+					<FormTextInput
+						id="trademarkNumber"
+						value={ trademarkNumber }
+						type="number"
+						autoCapitalize="off"
+						autoComplete="off"
+						autoCorrect="off"
+						placeholder={
+							translate( 'ex. 123456789',
+								{ comment: 'ex is short for example. The number is the EU trademark number format.' }
+							)
+						}
+						onChange={ this.handleChangeEvent } />
+				</FormFieldset>
+			</div>
+		);
+	}
+
+	renderOptional = () => {
+		return (
+			<span className="registrant-extra-info__optional-label">{ this.props.translate( 'Optional' ) }</span>
+		);
+	}
+}
+
+RegistrantExtraInfoForm.displayName = 'ExtraInfoFrForm';
```

Comment:
I haven't seen any internal announcements about this, but it was part of this massive PR for upgrading to Babel 6 long time ago (Dec 2015): https://github.com/Automattic/wp-calypso/pull/1515. Official docs: https://babeljs.io/docs/plugins/transform-react-display-name/ (not very helpful :/).

I wonder why it did not work in that case. Maybe there was no explicit variable involved?
```
// Will _not_ work:
export default class X extends Component { }

// Works:
class X extends Component {}
export default X;
```